src/test/run-make/export-executable-symbols/main.rs

```diff
@@ -0,0 +1,10 @@
+// edition:2018
+
+fn main() {
+    foo();
+}
+
+#[no_mangle]
+pub extern "C" fn foo() -> i32 {
```

Comment:
Don't we need to write a `Makefile` that checks the flag has some kind of effect?